lib/content-store/file-handler.js

```diff
@@ -0,0 +1,83 @@
+const path = require('path');
+const fs = require('fs');
+
+const BASEFILEPATH = path.resolve(__dirname, '..', '..', 'content');
+
+function loadFile(filepath) {
+  let filecontents;
+
+  try {
+    fs.accessSync(filepath, fs.F_OK);
+    filecontents = fs.readFileSync(filepath, 'utf8').trim();
+  } catch (e) {
+    filecontents = false;
```

Comment:
Should it be `null`? The other value is a string so it makes sense to present the absence of that value as `null`.